examples/rich/rich.html

```diff
@@ -89,6 +89,17 @@
             }
           }
 
+          var customBlockMap = Object.assign({}, DefaultDraftBlockRenderMap, {
+            'paragraph': {
+                wrapper: null,
+                element: 'p'
```

Comment:
Sorry to nitpick: FB standard is 2 spaces for indentation. (I guess the linter doesn't look at these.)